requirements.txt

```diff
@@ -66,4 +72,4 @@ werkzeug==0.16.1
 # It is not included by default because of the GPL license conflict.
 # ldap3==2.2.4
 Authlib==0.15.5
-advocate==1.0.0
\ No newline at end of file
+advocate==1.0.0
```

Comment:
Did you just move this further down?

Reply:
Nope -- this is just a new line. I'll remove it as it's not needed.